src/lowered.jl

```diff
@@ -33,6 +33,9 @@ function is_some_include(@nospecialize(f))
             f = f.value
         end
         if isa(f, Function)
+            if isdefined(Base, :IncludeInto) && isa(f, Base.IncludeInto)
```

Comment:
```suggestion
            if @static isdefined(Base, :IncludeInto) && isa(f, Base.IncludeInto)
```

Reply:
That changes doesn't seem necessary, but might as well have it (if further edits get made it might theoretically avoid some invalidation risk since `Base.IncludeInto` gets inferred as `Any` on existing Julias).